### DISCUSSION TITLE
Preselect the default language when creating a new translation string

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17135] [Feature Requests]
When creating a translation string, it would become handy to have the default language preselected. It is quite time consuming to select it every time. (Would also help to limit the languages to the ones who are actually used in the system. This would mean to provide an option in the settings to select the languages that are needed.)

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
